### PR TITLE
fixed inputs for ward models available for all the models

### PR DESCRIPTION
**Pull request by elisehuard**
have fixed inputs assigned to the right models

### CODE REVIEW

File: src/witan/app/test_data.clj [hunk 1]
```diff
@@ -289,8 +293,76 @@
                                                  :public? true
                                                  :s3-key #uuid "4b0d668f-e6b4-4f2c-bbaf-115819c21478"})
 
+        ;; - Ward level inputs
+        ward-php (data/add-data! {:category "ward-php"
```
Comment (sunnytownsend) [inline src/witan/app/test_data.clj]: please can you rename this to ward-adults-php, to distinguish from php data coming in elsewhere

Comment (elisehuard) [inline src/witan/app/test_data.clj]: will do

File: src/witan/app/test_data.clj [hunk 2]
```diff
@@ -310,13 +382,27 @@
...
+                                                 (data/->Data future-development-data)
+                                                 (data/->Data high-fert-principal-births-data)
+                                                 (data/->Data high-fert-principal-deaths-data)
+                                                 (data/->Data high-fert-principal-sya-data)
+                                                 (data/->Data standard-fert-principal-births-data)
+                                                 (data/->Data standard-fert-principal-deaths-data)
+                                                 (data/->Data standard-fert-principal-sya-data)
+                                                 (data/->Data low-fert-principal-births-data)
+                                                 (data/->Data low-fert-principal-deaths-data)
+                                                 (data/->Data low-fert-principal-sya-data)]}
```
Comment (sunnytownsend) [inline src/witan/app/test_data.clj]: These principal migration births, deaths and sya feed into the trend-based ward model only

File: src/witan/app/test_data.clj [hunk 3]
```diff
@@ -347,7 +433,12 @@
                                                         (data/->Data low-fert-high-sya-data)
                                                         (data/->Data low-fert-low-births-data)
                                                         (data/->Data low-fert-low-deaths-data)
-                                                        (data/->Data low-fert-low-sya-data)])))))
+                                                        (data/->Data low-fert-low-sya-data)
+                                                        (data/->Data population-data)
+                                                        (data/->Data institutional-data)
+                                                        (data/->Data private-housing-data)
```
Comment (sunnytownsend) [inline src/witan/app/test_data.clj]: Please can you change this to private-housing-popn-data

**Comment by sunnytownsend** [comment]
lgtm :shipit:

**Pull request merged**